### PR TITLE
feature(tape): Whiskey is dead. Use a modern testing tool.

### PR DESCRIPTION
**Pull request by jirwin**
You still use `npm test` to run the tests.

```
1..299
# tests 299
# pass  299

# ok
```

### CODE REVIEW

**Comment by robert-chiniquy** [comment]
<3

**Comment by gdusbabek** [comment]
That was a lot of busy work. +1. Thanks for this.

File: tests/test_metric_regrouping.js [hunk 1]
```diff
@@ -72,90 +75,90 @@ function arrayReduce(dict, element) {
...
     visitCount += 1;
   });
-  
-  assert.strictEqual(2, visitCount);
-  
-  test.finish();
-}
 
-exports['test_timers'] = function(test, assert) {
+  t.strictEqual(2, visitCount, 'Each set was checked.');
```
Comment (GeorgeJahad) [inline tests/test_metric_regrouping.js]: The original code uses assert.strictEqual().  Here you use t.strictEqual(), other places t.Equal().  I assume it doesn't matter, but was there a reason?

Comment (jirwin) [inline tests/test_metric_regrouping.js]: I had meant to make them all `t.equal()`. `t.strictEqual()` is an alias for it though.

**Comment by GeorgeJahad** [comment]
+1

**Pull request merged**